articles/native-platforms/android/07-authorization.md

```diff
@@ -0,0 +1,64 @@
+ ---
+ title: Authorization
+ description: This tutorial will show you how to use the Auth0 authentication API in your Android project to create a custom login screen.
+ seo_alias: android
+ ---
+ 
```

Comment:
@tenmaster The title does not display correctly:
![image](https://cloud.githubusercontent.com/assets/11715799/16835028/3b58e478-49bf-11e6-9286-1522f50fd6e3.png)


The reason is the space that exists before each line, remove it:
```
---
title: Authorization
description: This tutorial will show you how to use the Auth0 authentication API in your Android project to create a custom login screen.
seo_alias: android
---
```